src/utils/types.ts

```diff
@@ -1,92 +0,0 @@
-/*
```

Comment:
all of these mangled types are conflicting with the matching types of @types/eslint or @typescript-eslint/experimental-utils, so I replaced all of them with the types coming from those packages instead.